substrate/client/transaction-pool/src/builder.rs

```diff
@@ -224,7 +224,13 @@ where
 
 	/// Creates an instance of transaction pool.
 	pub fn build(self) -> TransactionPoolHandle<Block, Client> {
-		log::info!(target:LOG_TARGET, " creating {:?} txpool {:?}/{:?}.", self.options.txpool_type, self.options.options.ready, self.options.options.future);
+		tracing::debug!(
```

Comment:
I'd prefer `info` here, knowing which pool was create is kind of usefull.
Debug is only printed once, so I let's keep it.

```suggestion
		tracing::info!(
```